man1/aur-sync.1

```diff
@@ -162,6 +162,11 @@ Default: \fI$XDG_CACHE_HOME/aurutils/sync\fR
 Parent directory for temporary files.
 .RE
 
+.B AURURI
+.RS
+Sets the default URI for fetching packages with \fIaur-rpc\fR
```

Comment:
`aur-rpc` queries AurJson, fetching is done by `aur-fetch`. I would keep the description generic here, "Sets the default AUR URI" or such.